core/types/state_account.go

```diff
@@ -46,8 +46,15 @@ var (
 
 type isMultiCoin bool
 
-var IsMultiCoinPayloads = ethtypes.RegisterExtras[isMultiCoin]()
+var (
+	extras = ethtypes.RegisterExtras[
+		ethtypes.NOOPHeaderHooks, *ethtypes.NOOPHeaderHooks,
+		ethtypes.NOOPBlockBodyHooks, *ethtypes.NOOPBlockBodyHooks,
+		isMultiCoin,
+	]()
+	IsMultiCoinPayloads = extras.StateAccount
```

Comment:
```suggestion
```

You've modified `IsMultiCoin()` to use `extras` directly so I suspect this is now unnecessary.